src/app/shared/theme-picker/theme-picker.ts

```diff
@@ -82,7 +90,7 @@ export class ThemePicker implements OnInit, OnDestroy {
     this._queryParamSubscription.unsubscribe();
   }
 
-  installTheme(themeName: string) {
+  onThemeSelection(themeName: string) {
     const theme = this.themes.find(currentTheme => currentTheme.name === themeName);
 
     if (!theme) {
```

Comment:
`MatGridListModule` probably doesn't need to be imported into this module anymore.